src/viewport.js

```diff
@@ -198,6 +198,14 @@ export class Viewport {
   }
 
   /**
+   * Returns the scroll height of the content within the viewport.
+   * @return {number}
+   */
+  getScrollHeight() {
+    return this.binding_.getScrollHeight();
```

Comment:
Right now we can't because we have no events that help identify when we can cache. Unlike with "size" where the events are very clear. They are used, however, very sparingly. Even in this case (scroll adjustment for height changes), I anticipate that the probability of calls should be very small.